[2025-05-06T13:32:23.669+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2025-05-06T13:32:22.250962+00:00 [queued]>
[2025-05-06T13:32:23.675+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2025-05-06T13:32:22.250962+00:00 [queued]>
[2025-05-06T13:32:23.675+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2025-05-06T13:32:23.676+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2025-05-06T13:32:23.676+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2025-05-06T13:32:23.683+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2025-05-06 13:32:22.250962+00:00
[2025-05-06T13:32:23.685+0000] {standard_task_runner.py:55} INFO - Started process 21934 to run task
[2025-05-06T13:32:23.687+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2025-05-06T13:32:22.250962+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpc9qv5y0v']
[2025-05-06T13:32:23.688+0000] {standard_task_runner.py:83} INFO - Job 47: Subtask is_api_available
[2025-05-06T13:32:23.720+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2025-05-06T13:32:22.250962+00:00 [running]> on host 081a24b4289c
[2025-05-06T13:32:23.752+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2025-05-06T13:32:22.250962+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-05-06T13:32:22.250962+00:00
[2025-05-06T13:32:23.752+0000] {http.py:120} INFO - Poking: api/
[2025-05-06T13:32:23.757+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2025-05-06T13:32:23.758+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2025-05-06T13:32:24.071+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2025-05-06T13:32:24.078+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20250506T133222, start_date=20250506T133223, end_date=20250506T133224
[2025-05-06T13:32:24.113+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2025-05-06T13:32:24.144+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
